app/views/projects/_category-info.html.erb

```diff
@@ -5,6 +5,7 @@
   }
 
   # We need to explicitly write the full color instead of interpolating it so tailwind can later safely purge the unused css classes.
+  # Refactor?
```

Comment:
Hey @adrianthedev - I was asked to refactor this switch statement, but I can see you have left a comment here to leave it as is - is that still the case? If so, would you mind explaining why? Just so I can level up my front-end/tailwind knowledge 😁 

If this is no longer the case, then I will refactor to use string interpolation to isert the correct `bg-color` into the html

Reply:
Yeah, sure.
Tailwind works by taking in the config file and generating a loooot of CSS (classes). Like all the permutations possible. Every color, every size, each for every breakpoint available, etc.
In reality, you never use all of them. You only use a subset of them. So, in production there's a step in the build step (purgecss) that strips away the classes you don't use. In order to do that it needs to read all of your files and make a list (whitelist) of classes you use and discard the ones you don't use in the final built css file. It will not be able to read something like `bg_color = "bg-#{@project_category[:color]}-600"`. So you need to specify the full color name.
An alternative to this is to do the refactor using variables like I mentioned above and have all the used classes commented out somewhere in the file so purgecss can read them on compile time.

Reply:
Now that I inspected the code, HW really doesn't have that purgecss step enabled 🙃.
I pushed some code to fix that.

[`94c86d0` (#290)](https://github.com/helpwithcovid/covid-volunteers/pull/290/commits/94c86d09d219c6e4abee0c42c17dd2ef027b0ea0)

Reply:
This should be pushed to the master branch too. The final build payload will go on a diet after this.

Reply:
Coolio, and thanks for the explanation! I'll leave the switch statement as is then 👍  I'll also make a note to make a PR for `master` to add this purge step